doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -220,40 +222,49 @@ An B<ML-KEM-768> key can be generated like this:
 
 Equivalent calls are available for B<ML-KEM-512> and B<ML-KEM-1024>.
 
-An B<ML-KEM> private key in seed format can be converted to a key in expanded
-FIPS 203 B<dk> format by running:
+An B<ML-KEM> private key in seed format can be converted to a key in the FIPS
+203 B<dk> format by running:
 
-    $ openssl pkey -provparam ml-kem.retain_seed=no -in seed.pem -out long.pem
+    $ openssl pkey -provparam ml-kem.retain_seed=no \
+        -in seed-only.pem -out priv-only.pem
 
-To generate an, e.g., B<ML-KEM-768> key, in expanded format, you can run:
+To generate an, e.g., B<ML-KEM-768> key, in FIPS 203 B<dk> format, you can run:
 
     $ openssl genpkey -provparam ml-kem.retain_seed=no \
         -algorithm ml-kem-768 -out long.pem
 
+If you have B<PKCS#8> file with both a seed and a key, and prefer to import the
+companion key rather than the seed, you can run:
+
+    $ openssl pkey -provparam ml-kem.prefer_seed=no \
+        -in seed-priv.pem -out priv-only.pem
+
 In the B<openssl.cnf> file, this looks like:
 
- openssl_conf = openssl_init
+    openssl_conf = openssl_init
 
- [openssl_init]
- providers = providers_sect
+    [openssl_init]
+    providers = providers_sect
 
- # Can be referenced in one or more provider sections
- [ml_kem_sect]
- retain_seed = yes
- # OQS legacy formats disabled
- input_formats = seed-priv, seed-only, priv-only
- # Output either the seed alone, or else the key alone
- output_formats = seed-only, priv-only
+    # Can be referenced in one or more provider sections
+    [ml_kem_sect]
```

Comment:
I assume we could share the section with ml-dsa also?

Reply:
Yes, an observant user could in fact share the section:
```
[ default_sect ]
ml-kem = ml_section
ml-dsa = ml_section
```
and get both sets of variables set the same.